tmt-app/src/Components/QuestionCard.jsx

```diff
@@ -19,45 +18,41 @@ const QuestionCard = ({ item }) => {
     setIsSelected(!isSelected);
   };
 
-  const style = {
+  const sx = {
     alignItems: "center",
     borderRadius: "36px",
     boxShadow: "6px 6px 16px rgba(0, 0, 0, 0.16)",
     display: "flex",
-    flex: "none",
     flexDirection: "column",
-    flexGrow: 0,
     gap: "0.5rem",
+    height: "152px",
     justifyContent: "center",
     margin: "0.6rem",
-    minHeight: "152px",
-    padding: "1rem",
+    maxWidth: "240px",
     textAlign: "center",
     transition: "all 0.3s ease-in-out",
-    width: "240px",
   };
 
   if (isSelected) {
-    style.border = "none";
-    style.background = CustomTheme.palette.ThaiTea.main;
-    style.color = "white";
+    sx.border = "none";
+    sx.background = CustomTheme.palette.ThaiTea.main;
+    sx.color = "white";
   } else {
     if (isHovered) {
-      style.border = `12px solid ${CustomTheme.palette.RedBean.main}`;
-      style.background = "white";
-      style.color = "black";
+      sx.border = `12px solid ${CustomTheme.palette.RedBean.main}`;
+      sx.background = "white";
+      sx.color = "black";
     } else {
-      style.border = "none";
-      style.background = CustomTheme.palette.BobaHighlight.main;
-      style.color = "white";
+      sx.border = "none";
+      sx.background = CustomTheme.palette.BobaHighlight.main;
+      sx.color = "white";
     }
   }
 
   return (
-    <Grid
-      item
+    <Box
       key={item.title}
-      style={style}
+      style={sx}
```

Comment:
Fyi, using a variable named `sx` won't be the same as using the `sx` prop available through MUI components https://mui.com/system/getting-started/the-sx-prop/